MOBILE_CLIENT/ANDROID/_MEDIUM/APK_INSECURE_PACKAGE_CONTEXT/description.md

```diff
@@ -0,0 +1,3 @@
+The vulnerability is related to the use of the createPackageContext method with the CONTEXT_INCLUDE_CODE and CONTEXT_IGNORE_SECURITY flags 
```

Comment:
The description is not clear enough. How can it lead loading and execution of arbitrary code ?
Also, use markdown to highlight the name of the method and the flags